gcs-action/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "gcs-action",
+  "version": "0.0.1",
+  "private": true,
+  "description": "GCS action",
+  "main": "lib/main.js",
+  "scripts": {
```

Comment:
Please match the scripts from https://github.com/GoogleCloudPlatform/github-actions/pull/60 (sorry for the churn).

Reply:
Done